### PR TITLE
ENH: Add welcome message that data stays local to the machine

### PR DESCRIPTION
**Pull request by aylward**

### CODE REVIEW

File: src/components/App.vue [hunk 1]
```diff
@@ -121,11 +121,18 @@
...
                         <v-icon size="64">mdi-folder-open</v-icon>
                       </div>
-                      <div>Click anywhere here to open files</div>
+                      <div>Click to open local files.</div>
+                      <br><br>
                       <div class="mt-8">
                         <v-icon size="64">mdi-arrow-down-bold</v-icon>
                       </div>
-                      <div>Drop your files anywhere here to open</div>
+                      <div>Drag & drop your DICOM files.</div>
```
Comment (floryst) [inline src/components/App.vue:129]: `&` should be `&amp;`

File: src/components/App.vue [hunk 2]
```diff
@@ -121,11 +121,18 @@
...
+                      <br><br><br><br>
```
Comment (floryst) [inline src/components/App.vue:130]: I think these `<br>` tags are unnecessary. You can bump up the spacing by changing `mt-8` classes to `mt-9` or `mt-10`.

Comment (aylward) [reply src/components/App.vue]: Fixed.

File: src/components/App.vue (hunk 1 above)
Comment (aylward) [reply src/components/App.vue]: Fixed.

**Comment by aylward** [comment]
Replaced by https://github.com/KitwareMedical/VolView/pull/215

**Pull request closed without merging**